HW6/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+#thanks to Will for help on getting the yml file working
+
+version: "3.3"
+services:
+
+  hw2:
+    container_name: hw2
+    depends_on:
+      - db-service
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./results:/results
+    networks:
+      - backend
+
+  db-service:
+    image: linuxserver/mariadb:latest
```

Comment:
I like how you use latest here, I assume it will continue to work when versions are updated